app/view_objects/daily_statistic_notifier.rb

```diff
@@ -0,0 +1,33 @@
+class DailyStatisticNotifier
```

Comment:
@vlasiak, I do not think this is a notifier. I would say that it is a Daily ToDo Statistic or Daily ToDo Digest.